### ISSUE TITLE
error: Package `dash-' is unavailable

### ISSUE THREAD

**Issue by neverfox** [ardumont/prelude-live-pack#1]
Hopefully this is the last little thing before getting emacs-live-packs back on track. Everything is running smoothly until I add this pack. I get the following in the _Warnings_ bufffer (and emacs-live gives me the mushroom cloud):

```
error: Package `dash-' is unavailable
```

``` elisp
(emacs-live-packs/add-live-packs "~/.emacs-live-packs/"
                                 '("install-packages-pack"
                                   ;;"theme-pack"
                                   ;;                                   "el-get-pack"
                                   "buffer-pack"
                                   "scratch-pack"
                                   ;;"blog-pack"
                                   ;;"haskell-pack"
                                   ;;"orgmode-pack"
                                   "lisp-pack"
                                   "git-pack"
                                   ;;"mail-pack"
                                   "shell-pack"
                                   ;;"browser-pack"
                                   ;;"chat-pack"
                                   "clojure-pack"
                                   ;;                                   "nrepl-pack"
                                   "clojurescript-pack"
                                   ;;"caml-pack"
                                   "modeline-pack"
                                   ;;"twitter-pack"
                                   ;;"puppet-pack"
                                   ;;                                   "chrome-pack"
                                   ;;"macro-pack"
                                   ;;"scala-pack"
                                   "elisp-pack"
                                   ;;"groovy-pack"
                                   ;;"php-pack"
                                   ;;                                   "ctags-pack"
                                   "prelude-live-pack"
                                   ;;"stumpwm-pack"
                                   ;;"pres-pack"
                                   ))
```

**Comment by ardumont** [comment]
Hello,

I'm sorry I did not get which pack you added?
This will help me solve this fast :D

**Comment by ardumont** [comment]
Is it "buffer-pack"?

**Comment by ardumont** [comment]
lol.
I'm tired, it's prelude-live-pack, isn't it? :D

**Comment by neverfox** [comment]
Yes ;)

**Comment by ardumont** [comment]
Of course it won't find `dash`.
`dash` is not a default package lib.
We need to have at least melpa or marmalade repositories first... to be able to install it.

My fault is that prelude already uses melpa and thus do not break on this.
As for emacs-live, it does not use package...
Thus the break!

Working on the solution.

**Comment by ardumont** [comment]
It was a problem in `install-packages-pack` really.

Should have been fixed now.
Update emacs-live-packs + install-packages-pack and you should be good to go.
[Also the fix I asked you to add in your ~/.emacs-live.el is no longer necessary](https://github.com/ardumont/install-packages-pack/issues/1#issuecomment-104047886)

Cheers,

**Comment by neverfox** [comment]
Thanks for all of your help with this. I'm now getting the following:

```
Wrong type argument: symbolp, ((gnu . http://elpa.gnu.org/packages/))
```

**Comment by ardumont** [comment]
Hello,

> Thanks for all of your help with this. I'm now getting the following:
>     Wrong type argument: symbolp, ((gnu . http://elpa.gnu.org/packages/))

Thanks for your patience, I made a typo just after having tested and before submitting the patch...

I forgot to add back a quote to [package-archives'  form](https://github.com/ardumont/install-packages-pack/blob/master/install-packages-pack.el#L26)...
(The story is to eval a form within emacs using `C-M-x`, the form should not be quoted... otherwise, it does not work...)

Anyway, I fixed and tested it again.
- creating the right links for my emacs install to be emacs-live
- removing `~/.emacs.d/elpa` folder to let the initialization takes place
- started emacs-live
- did not touch anything after that 
- pushed the fix

To sum up, it's now fixed.
Update your `emacs-live-packs` and `install-packages-pack` again.
And fire up the live beast :D 

Cheers,

**Comment by ardumont** [comment]
Hello,

Is this now ok for you?

Cheers,

**Comment by neverfox** [comment]
Yes, indeed. In fact, it's the only pack loading given the other issue you are working on with the whole loading process, since it's still in the old format.
